core/themes/basis/css/component/fieldset.css

```diff
@@ -100,21 +100,27 @@ fieldset .fieldset-wrapper {
 }
 
 .fieldset-legend span.summary {
-  position: absolute;
-  top: .4em;
-  right: 2em;
+  position: relative;
```

Comment:
As an alternative for switching from position absolute to relative in combination with float, you might be able to achieve the same with

```
[dir="rtl"] .fieldset-legend span.summary {
  right: auto;
  left: 2em;
}
```
Then you'd also not need the negative top margin, I suppose. Didn't actually test it, though, it just seems a smaller change to me. Worth a try?

Reply:
@indigoxela and @jenlampton they way you suggest to change this is how the respective selector is set in Seven. This is Basis though, and this is what it looks like if I try your suggestion:

![image](https://user-images.githubusercontent.com/2423362/131487102-db76bcb0-a42c-4b83-8357-6666c4d13637.png)

...my code fixes that, but I don't claim to be a CSS expert. Can either of you please file a PR against mine if you still think that it should be fixed differently, or suggest other CSS that might work better? Thanks.

Reply:
...I fiddled a bit with the CSS, and I managed to make it work with `position: absolute;` as before.

Reply:
@klonos oops, you ripped off the sandbox I was just playing with. :laughing: 

What I suggested - really only a suggestion:

```
.fieldset-legend span.summary {
  position: absolute;
  top: .7em;
  color: #999;
  right: .5em;/* LTR */
}

[dir="rtl"] .fieldset-legend span.summary {
  right: auto;
  left: 2em;
}
```
Not sure, if this is complete.